app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: "from@example.com"
```

Comment:
would advice getting rid of this, possibly replacing with what you have set in user_mailer.rb since it seems like a sensible default from option